src/SerializedHighlight.ts

```diff
@@ -83,6 +84,12 @@ export default class SerializedHighlight {
     const prevHighlight = highlighter.getHighlightBefore(highlight);
     const nextHighlight = highlighter.getHighlightAfter(highlight);
 
+    let contentPath: number[] | undefined;
+
+    if (serializationData.type === xPathDiscriminator) {
```

Comment:
I'm testing out some changes to this to use text position. I'm not sure if there's a case where the strategies could get mixed, but my guess is no, and that it's safe to say comparisons would always be the same type for a given reference id. Since xpath is the default it's hard for me to understand how text position gets used in real content (though how it is represented is very simple.)

Reply:
Okay I checked today and it looks like Tutor is using Xpath not TextSelector, so I'm not sure of a "real" test. But I think this approach here will work, since the highlighted text parent should always be the reference element, so just return 0 and the offset: https://github.com/openstax/highlighter/pull/61/commits/6cf0fd36aef1d883872143101c4257ab8569f076#diff-db95139714bd7e6e07fffb3e5099c11f8fcb08d2263985401647db6d2c6e8f15R10